examples/game/breakout.rs

```diff
@@ -102,11 +104,11 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
             },
             ..default()
         })
-        .insert(Paddle {
-            speed: PADDLE_SPEED,
-        })
-        .insert(Collider::Paddle);
+        .insert(Paddle)
```

Comment:
This is a really minor nit and totally up to you -- I tend to insert the "what this thing is" marker component first so that the entity being spawned is readily identifiable even without comments.